### DISCUSSION TITLE
How are you redistributing the Python interpreter, libs, etc.?

### DISCUSSION THREAD

**Discussion by jdluzen** [pythonnet/pythonnet#2023] [Q&A]
I'm attempting to package and redistribute Python with my .NET7 Windows app. I've tried a number of packagers including py2exe, pyInstaller, and Nuitka. Every one has had different errors one way or another, even after diving down rabbit holes trying to adjust configuration files, include additional imports, etc. How are you all redistributing Python with your apps?

**Comment by CatBraaain** [comment]
Today I successed compiling python with below commands via pyinstaller and nuitka
```bat
uv run pyinstaller --onefile --console test.py
uv run nuitka --mode=onefile test.py --mingw64 --output-dir=dist
```
